<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>685</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>jf open 粉圓 1.1</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(63, 63, 63)</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="filterWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>240</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(49, 49, 49);
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="centerVerticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="searchWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>7</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>7</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="searchDisplayLabel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微軟正黑體</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color:rgb(195, 195, 195)</string>
               </property>
               <property name="text">
                <string>請輸入要下載的圖串</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="searchBarLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="searchTextEdit">
                 <property name="font">
                  <font>
                   <family>jf open 粉圓 1.1</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(195, 195, 195);
border-style: solid;
border-color: rgb(149, 149, 149);
border-width: 2px;
border-radius: 6px;</string>
                 </property>
                 <property name="placeholderText">
                  <string>請輸入巴哈姆特文章串...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="searchButton">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>jf open 粉圓 1.1</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color:rgb(73, 57, 255);
background-color:rgb(255, 247, 137);
border-style: solid;
border-radius: 6px;</string>
                 </property>
                 <property name="text">
                  <string>搜尋</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="searchDetailLayout">
               <item>
                <widget class="QCheckBox" name="clearPreSeachCheckBox">
                 <property name="font">
                  <font>
                   <family>微軟正黑體</family>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color:rgb(195, 195, 195)</string>
                 </property>
                 <property name="text">
                  <string>清除前次搜尋結果</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="floorDisplayLabel">
                 <property name="font">
                  <font>
                   <family>jf open 粉圓 1.1</family>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color:rgb(195, 195, 195)</string>
                 </property>
                 <property name="text">
                  <string>樓層：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="floorTextEditStart">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>微軟正黑體</family>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(195, 195, 195);
border-style: solid;
border-color: rgb(149, 149, 149);
border-width: 2px;
border-radius: 6px;</string>
                 </property>
                 <property name="wrapping">
                  <bool>false</bool>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::NoButtons</enum>
                 </property>
                 <property name="keyboardTracking">
                  <bool>true</bool>
                 </property>
                 <property name="showGroupSeparator" stdset="0">
                  <bool>false</bool>
                 </property>
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="floorDisplayLabel_2">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>jf open 粉圓 1.1</family>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color:rgb(195, 195, 195)</string>
                 </property>
                 <property name="text">
                  <string> 至 </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="floorTextEditEnd">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>微軟正黑體</family>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(195, 195, 195);
border-style: solid;
border-color: rgb(149, 149, 149);
border-width: 2px;
border-radius: 6px;</string>
                 </property>
                 <property name="wrapping">
                  <bool>false</bool>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="buttonSymbols">
                  <enum>QAbstractSpinBox::NoButtons</enum>
                 </property>
                 <property name="keyboardTracking">
                  <bool>true</bool>
                 </property>
                 <property name="showGroupSeparator" stdset="0">
                  <bool>false</bool>
                 </property>
                 <property name="maximum">
                  <number>487630000</number>
                 </property>
                 <property name="value">
                  <number>999999</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="centerScrollArea">
          <property name="styleSheet">
           <string notr="true">border-style: solid;
border-color: rgb(149, 149, 149);
border-width: 1px;
border-radius: 16px;</string>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="centerScrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>832</width>
             <height>488</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>jf open 粉圓 1.1</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(195, 195, 195);
background-color:rgba(99, 99, 99, 192)</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>檔案</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
